src/systems/systems/src/simulation/update_context.rs

```diff
@@ -108,6 +128,22 @@ impl UpdateContext {
         self.longitudinal_acceleration
     }
 
+    pub fn lat_accel(&self) -> Acceleration {
+        self.lateral_acceleration
+    }
+
+    pub fn vert_accel(&self) -> Acceleration {
+        self.vertical_acceleration
+    }
+
+    pub fn pitch(&self) -> Angle {
+        self.pitch
+    }
+
+    pub fn bank(&self) -> Angle {
+        self.bank
```

Comment:
**thought**: Maybe you also want to do something funky like I suggested above here?

Reply:
Indeed just putting it in an array isn't very useful. I'd suggest a type:
```rust
struct Attitude {
    pitch: Angle;
    bank: Angle;
}
impl Attitude {
    fn pitch_rotation(&self) -> Rotation3<f64> { // Not quite sure about this function name, you probably have a better one!
        Rotation3::from_axis_angle(&Vector3::x_axis(), context.pitch().get::<radian>())
    }
}